app/controllers/events/view/sessions/list.js

```diff
@@ -29,7 +29,7 @@ export default Controller.extend({
     },
     {
       propertyName : 'last-modified',
-      template     : 'components/ui-table/cell/cell-simple-date',
+      template     : 'components/ui-table/cell/events/view/sessions/cell-last-modified-date',
```

Comment:
It is just a simple date. No customisation needed. So why do you need a dedicated template for it ? Make it work with `cell-simple-date`. So that the same template can be reused for all date columns

Reply:
@niranjan94 without any formatting the date would look like 
![image](https://user-images.githubusercontent.com/22127980/40554341-046dd1f0-6063-11e8-80e0-90efc3fc55c0.png)
and if I use the cell-simple-date template then only submitted at field is displayed as per the definition of the template. Is there something I am missing out?

Reply:
The templates should be as agnostic to the table as possible. The `cell-simple-date` template should be able to display any date given to it.

Reply:
The templates should be as agnostic to the table as possible. The `cell-simple-date` template should be able to display any date given to it. 

That is the whole point of passing in a `propertyName` to the column definition. Instead of hard-coding a property within the template, you should use the propertyName to retrieve it.

With something like `(get record column.propertyName)` in the template. Please make changes to `cell-simple-date` and all its usages accordingly and use that same template for the `Last Modified At` column too in this PR.   

---

After this PR is done, open an issue to investigate and fix any other wrong usages of cell templates in the frontend.